docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+.. To find out more about how Flocker manages migration of containers with volumes, see *add link here*
+
+Step 1: Installing the Flocker client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following in your terminal:
+
+OS X
+****
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+************
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+To test your installation, run the following to check that you have the Flocker client installed correctly:
+
+.. prompt:: bash [you@laptop:~$]
+
+   flocker-deploy --version
+
+Successful installation will display the version of Flocker.
+
+Step 2: Installing Flocker on your hosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Option A: Installing Flocker on local VMs
+*****************************************
+
+.. note:: You must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_  installed to create the VMs and start the containers for this tutorial.
+
+In Step 1 you installed the Flocker client on your local machine.
+For the next step in this tutorial you now need two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+To install Flocker (plus dependencies) on the hosts, run the following command and Vagrant will create the environments you need:
+
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key && \
+   vagrant up && \
+   [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
+   ssh-add ~/.vagrant.d/insecure_private_key
+
+Option B: Installing Flocker on AWS or Rackspace
+************************************************
+
+The two instances of Flocker each run on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+To install Flocker (plus dependencies), follow the links to the direct instructions:
+
+- :ref:`AWS install instructions <aws-install>`
+- :ref:`Rackspace install instructions <rackspace-install>`
+
+Step 3: Deploying an app on the first host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will now have the Flocker client installed on your local machine and two instances of Flocker, each on a different host.
```

Comment:
This makes it seem that the product is in two pieces:
* the Flocker client
* Flocker

But Flocker is the product, it has two pieces (with various terms):
* The CLI
* The node software